src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -458,11 +458,11 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
     new ApiResponse(code = 500, message = "Rawls Internal Error")
   ))
   def copyMethodConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
-    path("workspaces" / Segment / Segment / "methodconfigs" / "copy" ) { (workspaceNamespace, workspaceName) =>
+    path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment / "copy" ) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigurationName) =>
       post {
         entity(as[MethodConfigurationName]) { srcMethodConfigurationName =>
           requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
-            WorkspaceService.CopyMethodConfiguration(workspaceNamespace, workspaceName, srcMethodConfigurationName))
+            WorkspaceService.CopyMethodConfiguration(MethodConfigurationName(methodConfigurationNamespace,methodConfigurationName,WorkspaceName(workspaceNamespace, workspaceName)), srcMethodConfigurationName))
```

Comment:
I don't really like having either source or dest in the uri, seems arbitrary which you pick so why have either?

How about a root level /copy_methodconfig end point that receives a json structure that specifies both source and destination workspaces and method config names? Looking forward to copying from method repo, the source could also be a reference to the method repo.